Makefile

```diff
@@ -152,3 +153,9 @@ infoblox-secret:
 	kubectl -n ohmyglb create secret generic external-dns \
 	    --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=$${WAPI_USERNAME} \
 	    --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=$${WAPI_PASSWORD}
+
+.PHONY: test-round-robin
+test-round-robin:
+	@kubectl run -it --rm networkutils --restart=Never --image=yauritux/busybox-curl -- sh -c \
```

Comment:
Maybe it's better to use official curl image https://hub.docker.com/r/curlimages/curl ... Most probably yauritux is incredibly nice guy, still better to trust official upstream images

Reply:
I used that because `curlimages/curl` is running under curl_user , having no permissions to modify `/etc/resolv.conf`

Reply:
after slack talk with Yury we found solution